addons/binding/org.openhab.binding.sleepiq/src/main/java/org/openhab/binding/sleepiq/config/SleepIQCloudConfiguration.java

```diff
@@ -0,0 +1,26 @@
+/**
+ * Copyright (c) 2014-2016 by the respective copyright holders.
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ */
+package org.openhab.binding.sleepiq.config;
+
+/**
+ * Configuration class for the SleepIQ cloud.
+ *
+ * @author Gregory Moyer - Initial contribution
+ */
+public class SleepIQCloudConfiguration {
+
+    public static final String USERNAME = "username";
+    public static final String PASSWORD = "password";
+    public static final String POLLING_INTERVAL = "pollingInterval";
+    public static final String LOGGING = "logging";
```

Comment:
I will replace the logging config option with ```logger.isDebugEnabled()``` to align with project standards. I will also look into using a different JAX-RS client implementation for the upstream library, but that will likely be a future release.